webapp/views/partials/home.jade

```diff
@@ -1,4 +1,5 @@
 div(class='container', ng-if='init()')
+    -var selected = 'Home';
```

Comment:
So now with our "states" we can disallow home button being clicked on the home page, which we were having the problem before. Before, if in home and waiting for match/matched, clicking home button would navigate to state 'home' without re-evaluating controller. This cleared the sub state matching/matched. So not letting user click the button stops that problem :)